lib/helpers.js

```diff
@@ -0,0 +1,686 @@
+'use strict';
+
+const showdown = require('showdown');
+const Ansi = require('ansi-to-html');
+const Handlebars = require('handlebars');
+const util = require('util');
+const utils = require('./utils');
+
+/**
+    *	Get the value of an object's property
+    *	@param {object} The object
+    *	@param {string} The property's path
+    *	@example {{get object path}}
+    *   @returns The value of object's property or ""
```

Comment:
:warning:  Aca tmb quedo mal

```suggestion
    * @returns The value of object's property or ""
```

Reply:
Arreglado